lldb/source/Plugins/ExpressionParser/Swift/SwiftASTManipulator.cpp

```diff
@@ -1305,10 +1312,11 @@ swift::ValueDecl *SwiftASTManipulator::MakeGlobalTypealias(
   swift::TypeAliasDecl *type_alias_decl = new (ast_context)
       swift::TypeAliasDecl(swift::SourceLoc(), swift::SourceLoc(), name,
                            swift::SourceLoc(), nullptr, &m_source_file);
-  swift::Type underlying_type = GetSwiftType(type);
-  type_alias_decl->setUnderlyingType(underlying_type);
-  type_alias_decl->markAsDebuggerAlias(true);
-  type_alias_decl->setImplicit(true);
+  if (swift::Type underlying_type = GetSwiftType(type)) {
+    type_alias_decl->setUnderlyingType(underlying_type);
+    type_alias_decl->markAsDebuggerAlias(true);
+    type_alias_decl->setImplicit(true);
+  }
 
```

Comment:
Why do we not return a nullptr here? It sounds dangerous to return a dangling type alias with underlying type.

Reply:
Yup, you're right!